app/views/other-resources.html

```diff
@@ -11,14 +11,16 @@
           </div>
           <alerts alerts="alerts"></alerts>
           <div class="data-toolbar">
-            <ui-select ng-model="kindSelector.selected" theme="bootstrap" search-enabled="true" ng-disabled="kindSelector.disabled" title="Choose a resource" style="width: 250px;">
-              <ui-select-match placeholder="Choose a resource to list...">{{$select.selected.kind | humanizeKind : true}}</ui-select-match>
-              <ui-select-choices repeat="kind in kinds | filter : {kind: $select.search} : matchKind | orderBy : 'kind'">
-                <div ng-bind-html="(kind.kind | humanizeKind : true) | highlight: $select.search"></div>
-              </ui-select-choices>
-            </ui-select>
+            <div class="col-xs-4 col-sm-4 col-md-2 pad-left-none pad-right-none">
```

Comment:
I think this works here, but just a quick note. I tend to avoid mixing bootstrap classes w/classes that also affect margin/padding/float on the same DOM node.  The reason being that its helpful to keep bootstraps class behavior predictable for mobile resize, etc.

A quick example:

```html
<div class="row">
  <div class="col-md-5">
     <div class="my-class">Hello</div> <!-- nested 'my-class' can have margin,padding,etc w/o breaking the grid -->
  </div>
  <div class="col-md-7">
    <div class="my-other-class">World</div> <!-- same -->
  </div>
</div>
```